test/burials/config/claimantContactInformation.unit.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/burials/config/form.js';
+
+describe('Burials claimant information', () => {
+  const { schema, uiSchema } = formConfig.chapters.claimantContactInformation.pages.claimantContactInformation;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input, select, textarea').length).to.equal(8);
+  });
+  it('should show errors when required fields are empty', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+    submitForm(form);
+    expect(Array.from(formDOM.querySelectorAll('.usa-input-error')).length).to.equal(6);
+    expect(onSubmit.called).not.to.be.true;
+  });
+  it('should submit when all required fields are filled in', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          definitions={formConfig.defaultDefinitions}
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_claimantAddress_street'), {
```

Comment:
I added a helper to make this easier. In `schemaform-utils`, there's a `getFormDOM` function. If you replace `findDOMNode` with that, you can do `formDOM.fillData(id, value)`. There are examples in the marriage page tests for pensions.

(Don't feel like you have to convert this, but should make writing this kind of test easier in the future).